examples/graph_db/chat.py

```diff
@@ -103,28 +114,59 @@ def inquiry(client, package_name: str):
         return []
 
 
+def remove_database(client, db_name):
+    delete_query = """
+            MATCH (n)
+            DETACH DELETE n
+        """
+    if client.execute_write_query(delete_query):
+        print("[green]Database is deleted!")
+    else:
+        print("[red]Database is not deleted!")
+
+
 def chat(opts: CLIOptions) -> None:
     print(
         """
-        [blue]Welcome to Langroid Dependencies chatbot!
+        [blue]Welcome to Dependency Analysis chatbot!
         Enter x or q to quit at any point.
         """
     )
 
     load_dotenv()
 
     neo4j_cfg = Neo4jConfig(
-        uri="",
-        username="",
-        password="",
+        uri="neo4j+s://927d9aab.databases.neo4j.io",
+        username="neo4j",
+        password="N7UfdMmtjfWQhAAmf42q1FdBzXB5F2m-Nleey1pmv20",
```

Comment:
@pchalasani I already changed the password after realizing this, but not sure how to delete this from the commit. Where I already removed it in the next commit.